makeflow/src/dag_file.c

```diff
@@ -37,4 +61,24 @@ int dag_file_is_sink( const struct dag_file *f )
 		return 1;
 }
 
+int dag_file_exists( const struct dag_file *f )
```

Comment:
Just a clarification. This function is more a 'should exist', rather than an existence predicate, right?